packages/api-explorer/src/Doc.jsx

```diff
@@ -16,9 +18,7 @@ const Response = require('./Response');
 const ResponseSchema = require('./ResponseSchema');
 const EndpointErrorBoundary = require('./EndpointErrorBoundary');
 
-const Oas = require('./lib/Oas');
-const { Operation } = require('./lib/Oas');
-const getPath = require('./lib/get-path');
```

Comment:
Did you mean to delete this dependency? I see that `getPath` is still being referenced further down in this file... haven't fully traced so I very well may be wrong!

Reply:
Yep, `get-path` now exists in https://github.com/readmeio/oas and can be imported via `require('oas/utils').getPath`.

Reply:
See https://github.com/readmeio/oas/blob/master/utils.js

Reply:
perfect!

Reply:
merge away